src/medigan/generators.py

```diff
@@ -1351,6 +1352,11 @@ def get_as_torch_dataset(
             samples=samples, masks=masks, labels=labels, transform=transform
         )
 
+    def visualize(self, model_id: str):
```

Comment:
Also here, it would be good to have the docstring analogous to the other functions of the Generators() class.